packages/wrangler/src/index.ts

```diff
@@ -1486,6 +1486,280 @@ export function createCLIParser(argv: string[]) {
 	return wrangler;
 }
 
+/**
+ * EXPERIMENTAL: Create and return a command registry for documentation generation.
+ * This API is experimental and may change without notice.
+ */
+export function createCommandRegistry(): CommandRegistry {
```

Comment:
This is duplicative. Instead, modify `createCLIParser()` to also return `registry`, and use that directly